docs/install/pxe-boot-install.md

```diff
@@ -28,7 +28,7 @@ Let's assume an NGINX HTTP server's IP is `10.100.0.10`, and it serves `/usr/sha
 
 ## Preparing Boot Files
 
-- Download the required files from https://github.com/harvester/harvester/releases. Choose an appropriate version.
+- Download the required files from [Harvester Releases Page](https://github.com/harvester/harvester/releases). Choose an appropriate version.
```

Comment:
add a hyperlinks in 'pxe install' page because the original one is displayed as plain texts.